### PR TITLE
Update Forbes.com.xml

### PR DESCRIPTION
**Pull request by koops76**
#10782

### CODE REVIEW

**Comment by J0WI** [comment]
Maybe we need valid test urls here?

**Comment by koops76** [comment]
@J0WI for removed targets?

https://travis-ci.org/EFForg/https-everywhere/jobs/249090485

**Comment by J0WI** [comment]
Exactly.

**Comment by koops76** [comment]
@J0WI This error is exclusive to curl, any real browser works.

**Comment by J0WI** [comment]
The host are working fine for me using curl. If it's just a Travis thing we might exclude this from fetch tests.

**Comment by koops76** [comment]
@J0WI Maybe IP blocking.

**Comment by J0WI** [comment]
See https://github.com/EFForg/https-everywhere/pull/10944#issuecomment-313884041

**Comment by Hainish** [comment]
I don't see any `403` failures in https://travis-ci.org/EFForg/https-everywhere/jobs/249090485, I would say this is unrelated to #10944

**Comment by koops76** [comment]
@Hainish Maybe it's caused by incorrect `Content-Length` header value.

**Comment by koops76** [comment]
@Hainish https://stackoverflow.com/questions/1759956/curl-error-18-transfer-closed-with-outstanding-read-data-remaining

**Comment by J0WI** [comment]
Whitelist?

**Comment by Hainish** [comment]
Looks like tests are now passing.

**Comment by J0WI** [comment]
@koops76 could you please re-add the removed targets?

**Comment by koops76** [comment]
@J0WI Done.

File: src/chrome/content/rules/Forbes.com.xml [hunk 1]
```diff
@@ -1,34 +1,32 @@
...
-	<target host="esp.forbes.com"/>
+	<target host="esp.forbes.com" />
 	<target host="i.forbesimg.com" />
 	<target host="images.forbes.com" />
 	<target host="specials-images.forbesimg.com" />
 
-	<rule from="^http:" to="https:"/>
+	<securecookie host=".+" name=".+" />
+
+	<rule from="^http://www\.forbes\.com/" to="https://forbes.com/" />
```
Comment (J0WI) [inline src/chrome/content/rules/Forbes.com.xml]: Why is this redirect needed?

Comment (koops76) [inline src/chrome/content/rules/Forbes.com.xml]: @J0WI Removed.

**Comment by J0WI** [comment]
Let's add it to the whitelist.

**Comment by koops76** [comment]
@J0WI Done.

**Pull request merged**